src/main/java/me/lucien/minesweeper/web/GameController.java

```diff
@@ -8,14 +8,13 @@
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
 
 @RestController
 public class GameController {
 
     private Map<Integer, Room> roomMap = new HashMap<>();
+    private Map<Integer, Timer> timerMap = new HashMap<>();
```

Comment:
This makes GameController more complicated. We should add a new class RoomManager or something like that to manage room instances.